example/cumulus-tf/main.tf

```diff
@@ -45,10 +45,10 @@ module "cumulus" {
   prefix = var.prefix
 
   vpc_id            = var.vpc_id
-  lambda_subnet_ids = var.subnet_ids
+  lambda_subnet_ids = var.lambda_subnet_ids
 
   ecs_cluster_instance_image_id   = data.aws_ssm_parameter.ecs_image_id.value
-  ecs_cluster_instance_subnet_ids = var.subnet_ids
+  ecs_cluster_instance_subnet_ids = var.ecs_cluster_instance_subnet_ids
```

Comment:
The `example/cumulus-tf` module doesn't have a `ecs_cluster_instance_subnet_ids` input variable. Given that these tests are only run in NGAP, it's safe to leave this the way it was.

Reply:
ok